node/ChainwebNode.hs

```diff
@@ -470,7 +488,7 @@ mainInfo = programInfoValidate
 
 main :: IO ()
 main = do
-    installSignalHandlers
+    installFatalSignalHandlers [ sigHUP, sigTERM, sigXCPU, sigXFSZ ]
```

Comment:
I made `sigUSR2` not fatal anymore in case we want to use it later, so that we don't accidentally kill older versions of the node with it.

Reply:
Is `USER2` the only change compared to the previous behavior?

Reply:
sigUSR1 and sigUSR2 not being fatal is the only change.